src/ssort/_main.py

```diff
@@ -56,20 +58,29 @@ def main():
     for path in find_python_files(args.files):
```

Comment:
Might be worth adding something to `find_python_files` to prevent it from searching directories named `'-'`.

I _think_ I agree that it makes sense to allow mixing `'-'` with other arguments.